Documentation/Presentations.md

```diff
@@ -0,0 +1,224 @@
+# How to design and prototype custom presentation animation in Interface Builder with IBAnimatable
+
+For most of the non-trivial apps, we have to present modally some Scene (`ViewController`). August 20th, 2016 as of this writing, out of the box, Interface Builder only supports fullscreen modals with limited animations e.g. "Cover Vertical" and "Flip Horizontal". Using `IBAnimatable` we can configure in the interface builder the presentation animation, the dismissal animation, the modal size, the `dimmingView` appearance, and a lot more.
+
+
+## Configuring a Presentation in Interface Builder
+
+In order to configure a presentation in Interface Builder with `IBAnimatable`, you have to use `AnimatableModalViewController`
+
+Firstly, select a `UIViewController` then configure the custom class to `AnimatableModalViewController` in Identity inspector (![Identity inspector](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/IdentityInspector.png))
+
+![Transition - AnimatableViewController](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/PresentationUsingAnimatableViewController.png)
+
+Then we can configure the **Presentation Animations**, **Transition Duration** and **Modal specs** properties in Attributes inspector (![Attributes inspector](https://raw.githubusercontent.com/IBAnimatable/IBAnimatable-Misc/master/IBAnimatable/AttributesInspector.png)). 
+
+| Property | Description |
+| ------------- | ------------- |
+| Presentation Animation | The animation effect when *Present* a `ViewController`, you can find all supported transition animations in [Presentation Animators](#presentation-animators) section. |
+| Dismissal Animation | The animation effect when *Dismiss* a `ViewController`, you can find all supported transition animations in [Presentation Animators](#presentation-animators) section. By default, it will use the `Presentation animation` value |
```

Comment:
By default, it will use the `Presentation animation` value. For example, if the `Presentation animation` is slid down from the top, then the default `Dismissal Animation` is slid up to the top.